test/types/schema.test-d.ts

```diff
@@ -35,33 +35,6 @@ expectAssignable<FastifyInstance>(server.get(
   () => { }
 ))
 
-expectAssignable<FastifyInstance>(server.get<RouteGenericInterface, ContextConfigDefault, { validate:(data: any) => any }>(
```

Comment:
Why those tests were removed?

Reply:
These were removed as per @mcollina 's request. But happy to re-add them.

Reply:
@RafaelGSS Let me know if you would like me to re-add these tests.